### ISSUE TITLE
Gamesir app is not opening all of sudden

### ISSUE THREAD

**Issue by TengAhh** [gamesir123/app#94]
I tried to reinstall the app, restart my phone, kill process almost everything I tried but it never get fixed. I guess it's app to android os problem plz we need an update

**Comment by ed-jpa** [comment]
Yes, I've done all that too and still it does not open. I installed on another phone (redmi note 8 pro) and on that phone it opens! I don't understand. On mine, redmi note 10 pro it just doesn't open. Yesterday it worked fine when I installed for the first time. Tday it does not.

**Comment by TengAhh** [comment]
> Yes, I've done all that too and still it does not open. I installed on another phone (redmi note 8 pro) and on that phone it opens! I don't understand. On mine, redmi note 10 pro it just doesn't open. Yesterday it worked fine when I installed for the first time. Tday it does not.


Probably android12 os problem with the gamesir app. gamesir has to update their app

**Comment by Jballer1** [comment]
Install previous version of the app that works for me in on S22 Ultra with latest update.

**Comment by TengAhh** [comment]
Tried most of it already and 4.2.0 version only activates but that version doesn't work with android 12. useless try it was

**Comment by Jballer1** [comment]
I've been using 4.1.1, I'm using X2 usb type C, I connect the gamepad before opening the app then I start the game whiting the app.  And it works fine until you switch modes then the app crashes but I'm able to use the setup afterwards.
